src/lib/controller/message_handler.ts

```diff
@@ -102,11 +103,20 @@ export class ControllerMessageHandler {
         return { success };
       }
       case ControllerCommand.beginExclusion: {
-        const success = await driver.controller.beginExclusion(
-          message.strategy !== undefined
-            ? { strategy: message.strategy }
-            : undefined
-        );
+        let strategy: { strategy: ExclusionStrategy } | undefined;
+        if (message.strategy === undefined) {
+          strategy =
+            message.strategy !== undefined
```

Comment:
Something seems wrong here.

Reply:
@raman325 you may want to adopt what I did in the other PR before I saw this one:
https://github.com/zwave-js/zwave-js-server/pull/962/files#diff-9684276caee5caac11fdd0dd76254e6d8e3b9ed0072fe606d58381409711a43fR435-R455

That change essentially restores the functionality that was now removed from the driver.

Reply:
> Something seems wrong here.

This is what happens when I work on this while half asleep 🤦🏾 

> you may want to adopt what I did in the other PR before I saw this one

yes this is cleaner, thanks!